docs/.vuepress/config.js

```diff
@@ -1,6 +1,6 @@
 // .vuepress/config.js
 
-const MAIN_DOMAIN = process.env.MAIN_DOMAIN || 'https://staging.web3.storage'
+const MAIN_DOMAIN = process.env.MAIN_DOMAIN || 'https://web3.storage'
```

Comment:
```suggestion
const MAIN_DOMAIN = process.env.MAIN_DOMAIN || 'https://staging.web3.storage'
```

Let's leave this as is so in dev (local machines etc) we are pointing to the staging link, and I'll update the `MAIN_DOMAIN` var on fleek directly.

Reply:
`MAIN_DOMAIN` is set to `https://web3.storage` directly on fleek 👍

Reply:
We'll just all have to remember that's how it works on our local builds ... agree that it's best hygiene but we're not really doing that anywhere else 🤣

Reply:
Wait so the links I'm clicking are only going to the staging version of the main site because i'm accessing it locally, not because we made a magic feature to flip a switch with when the main site went live? Are we confident that the Spark team will continue to have their staging site in existence and identical to their live site? If so, I see where you're coming from for Countly reasons only; if not, I don't think I feel comfortable with this.

Reply:
On a related note, there are a few people on the team or files in the repo using the magical link forming method that makes this toggle work, but there are also a lot of straight up HTML references throughout that will only always go to the prod site. Do we care?

Reply:
> We'll just all have to remember that's how it works on our local builds ... agree that it's best hygiene but we're not really doing that anywhere else 🤣

True, in hindsight we don't have a particular reason to keep it as is I guess ahah.

> Wait so the links I'm clicking are only going to the staging version of the main site because i'm accessing it locally, not because we made a magic feature to flip a switch with when the main site went live?

The var `MAIN_DOMAIN` is set on the docs site in fleek. I changed it in fleek to the new prod url, so it overrides this local value in the `config.js` file.

Let's just do the change I guess, no harm done.

Reply:
> On a related note, there are a few people on the team or files in the repo using the magical link forming method that makes this toggle work, but there are also a lot of straight up HTML references throughout that will only always go to the prod site. Do we care?

Yes, it's not a perfect solution. This variable affects only UI components links, meaning header and footer links.